calendar-helper/index.js

```diff
@@ -24,7 +24,10 @@ class CalendarHelper {
         maxDate: null,
         ignoreWeekDays: [],
         shortcutButtons: [],
+        shortcutButtonsPlacement: 'top',
         hideIgnoredWeeks: false,
+        shortcutButtonCallbackSuffix: '',
+        shouldPrefixShortcutButtonCallbacks: false,
```

Comment:
Is it possible to have all the options under `shortcutButtons`? So that `shortcutButtons` can be an array (for backwards compatibility):
```
shortcutButtons: [],
```
or an object with the following options:
```
shortcutButtons: {
    buttons: [],
    placement: 'top',
    callbackSuffix: '',
    shouldPrefixButtonCallbacks: false,
}
```

I think this will ensure a clear interface.

Reply:
Fixed, readme updated